tests/Model/PHPProperty.php

```diff
@@ -38,15 +42,17 @@ public function readObjectFromReflection($reflectionObject): static
         }
         $this->access = $access;
         $this->is_static = $reflectionObject->isStatic();
-        $this->typesFromSignature = self::getReflectionTypeAsArray($reflectionObject->getType());;
+        if (doubleval(getenv('PHP_VERSION') >= 7.4)) {
```

Comment:
Maybe something like will be better: `method_exists($reflectionObject, 'getType')`?